pyproject.toml

```diff
@@ -43,3 +43,18 @@ flake8-eradicate = ["-*"]
 flake8-isort = ["+*", "-I001", "-I003", "-I004"]
 # don't fail on unknown docutils roles because sphinx knows them
 flake8-rst-docstrings = ["+*", "-RST304", "-RST303", "-RST301", "-RST203", "-RST201"]
+
+[tool.mypy]
+pretty = true
+python_version = 3.6
```

Comment:
Why 3.6? I'd start with 2.7 for the 8.x branch. Besides, it probably makes sense to test under different Pythons. WDYT?

Reply:
yes, mypy will just pickup the local interpreter

Reply:
I thought maybe we should run under multiple interpreters/platforms to cover the gaps.

Reply:
I think this is achievable via something like https://github.com/pypa/pip/blob/20.3.4/.pre-commit-config.yaml#L71-L80, at least partially.

Reply:
The issue is, how should we type then? The typing module is only there in 3.6 and even typing-extensions are only 3.6. Therefore it makes no sense to check for python 2. Or did I miss something?

Reply:
Use https://pypi.org/p/typing under Python 2.

Reply:
Thanks, I'll try that